cli-output/src/display.rs

```diff
@@ -28,6 +28,19 @@ impl Default for BuildBalanceMessageConfig {
     }
 }
 
+fn is_memo_program(k: &Pubkey) -> bool {
+    let memo_v1 = Pubkey::from_str("Memo1UhkJRfHyvLMcVucJwxXeuD728EqVDDwQDxFMNo")
```

Comment:
```suggestion
    // This will be available as spl_memo::v1::id() in v3.0.1 of spl-memo
    let memo_v1 = Pubkey::from_str("Memo1UhkJRfHyvLMcVucJwxXeuD728EqVDDwQDxFMNo")
```

https://github.com/solana-labs/solana-program-library/pull/1558